fatiando/seismic/conv.py

```diff
@@ -0,0 +1,126 @@
+"""
+
```

Comment:
@victortxa the first line of the module docstring should describe in a single sentence what this module does. In this case, a good examples would be "Seismic forward modeling by convolution" or "Zero-offset convolutional seismic modeling".

Next, put a small paragraph describing what the functions here do. Very general.

Finally, put a list of the functions present here with a line saying what each does (usually this is the first line of the function docstring).